app/assets/javascripts/service_calendar.js.coffee

```diff
@@ -164,35 +175,6 @@ $(document).ready ->
       data: data
   # NOTES LISTENERS END
 
-
```

Comment:
Removed javascript totals calculations and instead used HTML partials to render the costs on update.